doc/en/user/source/extensions/iau/install.rst

```diff
@@ -0,0 +1,23 @@
+.. _iauwkt.install:
+
+Installing the IAU authority
+----------------------------
+
+The IAU authority is an official extension.  Download the extension here - :download_extension:`iau`
+
+#. Download the extension for your version of GeoServer. 
+
+   .. warning:: Make sure to match the version of the extension to the version of GeoServer.
+
+#. Extract the archive and copy the contents into the GeoServer :file:`WEB-INF/lib` directory.
+
+#. Restart GeoServer.
+
+Verify Installation
+^^^^^^^^^^^^^^^^^^^
+
+To verify that the extension was installed successfully:
+
+#. Try to set up a layer (e.g., shapefile) with a IAU CRS (e.g., ``IAU:49900``, Mars (2015) - Sphere / Ocentric)
```

Comment:
I would of checked with the SRS demo page :)

> http:/localhost:8080/geoserver/web/wicket/bookmarkable/org.geoserver.web.demo.SRSDescriptionPage?3&code=IAU:49900

Easier test?

Reply:
Not implemented yet in the SRS page, it's still bound to the EPSG authority